superset/commands/database/export.py

```diff
@@ -100,9 +100,18 @@ def _export(
         payload["version"] = EXPORT_VERSION
 
         file_content = yaml.safe_dump(payload, sort_keys=False)
-        yield file_path, file_content
+        return file_content
+
+    @staticmethod
+    def _export(
+        model: Database, export_related: bool = True
+    ) -> Iterator[tuple[str, Callable[[], str]]]:
+        yield ExportDatabasesCommand._file_name(
+            model
+        ), lambda: ExportDatabasesCommand._file_content(model)
```

Comment:
Maybe use `functools.partial(ExportDatabasesCommand._file_content, model)` here for consistency?

Reply:
It is not required, because here we are making lambda function not in the loop.
In places where I use functools, I use it because of https://pylint.readthedocs.io/en/latest/user_guide/messages/warning/cell-var-from-loop.html.